addons/sitting/XEH_clientInit.sqf

```diff
@@ -7,6 +7,6 @@ if !(hasInterface) exitWith {};
 ["isNotSitting", {!((_this select 0) getVariable [QGVAR(isSitting), false])}] call EFUNC(common,addCanInteractWithCondition);
 
 // Handle interruptions
-["medical_onUnconscious", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetHandcuffed", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
-["SetSurrendered", {_this call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
+["medical_onUnconscious", {_player call DFUNC(handleInterrupt)}] call EFUNC(common,addEventhandler);
```

Comment:
Weird, worked just fine for me, I thought I'd remove number of arguments sending. On that note, are there any performance differences in how many arguments you are sending, even if so small it's not worth it?